libmscore/layout.cpp

```diff
@@ -1727,14 +1727,14 @@ void Score::respace(std::vector<ChordRest*>* elements)
 
 void LayoutContext::getNextPage()
       {
-      if (!page || curPage >= score->npages()) {
-            page = new Page(score);
-            score->pages().push_back(page);
+      if (!page || curPage >= dominantScore->npages()) {
```

Comment:
For me <code>dominantScore</code> is a strange name, I immediate have to think at a dominant chord. I understand it is the first score, so <code>firstScore</code> is a better name?
Or if this a kind of container, containing all the movements <code>albumScore</score> could be fine too.

Reply:
It is a terrible name. In the next few days I will make some drastic changes, then I will find a more suitable name :-)
`firstScore` or `firstMovement` seem decent.

Reply:
<code>firstMovement</code> seems the most descriptive name, indeed.